doc/process/jenkins_job_configuation_access.md

```diff
@@ -0,0 +1,74 @@
+# Introduction
+
+There are a number of cases where we would like working group members
+to be able to run and or edit jobs within jenkins. Examples include
+the CITGM jobs and the Benchmarking jobs.
+
+By default jobs can only be edited by members of the
+'nodejs/jenkins-admins' group which contains a subset of the build
+workgroup members. We do not give access to
+everbody to run/edit as it would be a potential security issue,
+both through the potential disclosure of secrets used as
+part of the builds, as well as a potential denial
+of service if jobs are launched maliciously.
+
+This document outlines the processes by which we will allow
+people outside the build working group to edit jobs as well
+as allowing people who are not collaborators run specific jobs.
+
+Unfortunately jenkins does not allow us to easily delegate the
+creation of new jobs in an appropriate manner.
+As such, the creation of a new job will
+need to be requested through an issue in the build repo. Any
+build member can then create a blank job in the group for the
+working group which can then be edited by those with access
+to the jobs for that working group. As and when more jobs are
+needed, a build member can clone one of the workgroup's 
+existing jobs, which will preserve the right permissions.
+
+## Ability to run/cancel jobs
+
+All members of a workgroup will be able to run and cancel
+the jobs tied to their workgroup.  This will be accomplished
+by enabling project-based security for the jobs and adding
+the group corresponding to the github team for the
+working group where the Build/Cancel rights have been
+enabled.
+
+## Ability to modify jobs
+
+A subset of the members of a workgroup will be able to modify
+and delete jobs. This will be accomplished by enabling
+project-based security for the jobs and adding the group
+corresponding to the github team for the working group admins
+where there Build/Cancel/Discover/Read/Delete/Update
+rights have been set.
```

Comment:
>This will be accomplished by enabling project-based security for the jobs and adding the group corresponding to the github team for the working group admins where there Build/Cancel/Discover/Read/Delete/Update rights have been set.

A bit hard to parse, maybe:

This will be accomplished by enabling project-based security for the jobs, and then giving the jenkins group corresponding to the github team for the workgroup admins Build/Cancel/Discover/Read/Delete/Update permissions.